### PR TITLE
adjusts burn modifiers of the baseline IPC and industrial IPC machine species... and prosthetics, by extension

### PR DESCRIPTION
**Pull request by OneOneThreeEight**
Changes: Industrial IPC burn modifier changed to 1.1. Baseline IPC burn modifier changed to 1.2. Industrial IPCs should tank more damage, baselines should tank less due to their more agile frame, though brute is still only moderately effective against baselines. They aren't brittle but aren't tanky like industrials. This also makes baselines take a fair bit more damage from remaining in a vacuum, which they took very little before. Industrials were applied this in a bit of an unfair way, however, given they are walking heat sinks and should be treated as such. Both will still get roasted by lasers, the baseline moreso than the industrial.

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
note to self: push upstream correctly and dont edit master, edit development for your own fork. will staple this onto my own forehead.

**Comment by Lohikar** [comment]
Smooth.

**Comment by OneOneThreeEight** [comment]
im going to kms

**Comment by Alberyk** [comment]
Except you are not taking in consideration that robotic limbs have reduce brute and burn damage by default.

**Comment by OneOneThreeEight** [comment]
anything else?

**Comment by OneOneThreeEight** [comment]
review me dad

**Comment by OneOneThreeEight** [comment]
@Lohikar reviewwwww

**Comment by skull132** [comment]
Pinged the IPC dev for lore review. Because. Lore.

**Comment by CakeIsOssim** [comment]
I approve.

**Comment by OneOneThreeEight** [comment]
@skull132 he approves

File: html/changelogs/1138-ipcburnmodpatch.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - balance: "Changes Baseline + Industrial burn modifiers. Baselines are more susceptible, Industrials are better heat sinks now."
+  - tweak: "Also changes default damage modifiers for synthetic limbs. Resist brute easier, melt easier by burns."
```
Comment (Lohikar) [inline html/changelogs/1138-ipcburnmodpatch.yml]: This is a bit awkwardly phrased.

Comment (OneOneThreeEight) [reply html/changelogs/1138-ipcburnmodpatch.yml]: Can't believe I actually typed that. must've been half asleep

**Pull request merged**